docs/4.3/enterprise/workflow/ssh_approval_jira_server.md

```diff
@@ -172,8 +194,7 @@ Here's the recommended Teleport Plugin service unit file for systemd:
 Save this as `teleport-jira.service`.
 
 ## Audit Log
-The plugin will let anyone with access to the Jira board so it's
-important to review Teleport's audit log.
+The plugin will let anyone with access to the Jira board so it's important to review Teleport's audit log.
```

Comment:
access to the Jira board so it's important -> access to the Jira board approve or deny requests, so it's important